setup.py

```diff
@@ -31,7 +31,7 @@
     'urllib3>=1.23',
     'requests>=2.0.1',
     'PySocks>=1.6.8',
-    'cryptography>=2.3',
+    'cryptography>=2.3,<=39.0.0',
```

Comment:
<40 should be ok — cryptography 39.0.2 is running fine on Archive-It's qa warcprox.

Reply:
PR updated, thank you.